pc_ble_driver_py/ble_adapter.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, events):
             self.conds[evt] = Condition(Lock())
         self.data = None
 
-    def wait(self, evt, timeout=5):
+    def wait(self, evt, timeout=10):
```

Comment:
@RyanLindemanCAE could you describe the reasoning behind increasing the default timeout value to 10? This would affect all calls to the EvtSync wait function, not just in the read_req function.

Reply:
I have been testing the original offset change in my older version of this driver but yesterday afternoon I tested this change with the latest version and discovered that my reads were failing until I made the above change. I realize that it does affect all events, not just read_req so I'm happy to drop this change and increase the read_req only version with a timeout of 10 if you prefer. I didn't see any other abnormal issues with this change though.